### DISCUSSION TITLE
Neovim toggleterm + gitui

### DISCUSSION THREAD

**Discussion by DartMitai** [extrawurst/gitui#1529] [Q&A]
Hi all sorry i'm noob for neovim and write code in terminal
when I am in neodim, a very cool toggleterm solution is used, but some gitui actions become unavailable, such as in this pop-up window, you need to press Esc to exit, which simply switches the terminal to another mode but does not exit.
![Снимок экрана от 2022-12-15 15-03-44](https://user-images.githubusercontent.com/104931116/207830843-d15ae49b-e5a4-4d41-b16b-43b2b3439777.png)
after pressing esc
![Снимок экрана от 2022-12-15 15-05-34](https://user-images.githubusercontent.com/104931116/207831179-89d35561-16ff-4e81-a31a-e87cc45c736d.png)
it can be closed only through the complete closure of the terminal, which is very inconvenient

**Comment by DartMitai** [comment]
I read about the keyboard settings, but I didn't understand anything, I didn't have this file in the folder .config/gitui/
there was only one file "theme.ron"
I made a file to configure the keys "key_bindings.ron" it's completely empty now, how do I change the exit button?
and I also couldn't activate the button of the third window Quit ^c I reset everything with the c button, but I didn't wait for the effect, what is it?

**Comment by extrawurst** [comment]
check `KEY_CONFIG.md` on the manual how to overwrite keys and `KeysList` inside `key_list.rs` shows you all the keys you and rebind. you will find that only `exit_popup` uses `esc` by default. so try rebinding this to `q`
